lib/standard/stream.nit

```diff
@@ -392,6 +392,29 @@ abstract class BufferedReader
 		return c.ascii
 	end
 
+	# Peeks up to `n` bytes in the buffer
+	#
+	# The operation does not consume the buffer
```

Comment:
Well, StringBuffer could become a BufferedReader, at a very small cost too, as long as we handle it properly, but I feel this would be outside of this PR's scope, maybe later !

And well, nitunits could be great, but until it becomes available on StringReader, they will fail constantly :/

I can always add examples as `nitish` however if you'd like